internal/display/members.go

```diff
@@ -0,0 +1,61 @@
+package display
+
+import (
+	"io"
+
+	"github.com/auth0/auth0-cli/internal/ansi"
+	"gopkg.in/auth0.v5/management"
+)
+
+type membersView struct {
+	ID         string
+	Name       string
+	Email      string
+	PictureURL string
+	raw        interface{}
+}
+
+func (v *membersView) AsTableHeader() []string {
+	return []string{"ID", "Name", "Email", "Picture"}
+}
+
+func (v *membersView) AsTableRow() []string {
+	return []string{ansi.Faint(v.ID), v.Name, v.Email, v.PictureURL}
+}
+
+func (v *membersView) KeyValues() [][]string {
+	return [][]string{
+		{"ID", ansi.Faint(v.ID)},
+		{"NAME", v.Name},
+		{"Email", v.Email},
```

Comment:
The keys for the key/value view should be in all caps. See https://github.com/auth0/auth0-cli/blob/main/internal/display/organizations.go#L33 as an example.